sass/_lagar-regler.scss

```diff
@@ -0,0 +1,40 @@
+.lagar-page {
+}
+h1 {
+  font-size: 3em;
+}
+h2 {
+  color: white;
+}
```

Comment:
these are global styles, put them inside lagar-page and they will only take effect on your page

Reply:
How do you mean? im not sure how to view the suggested changes in the PR so might need help fixing the code!

Reply:
They are not nested in the class that contains the content. When they are global they may change a h2 to white for me.